dodiscover/replearning/gin.py

```diff
@@ -0,0 +1,115 @@
+"""
+Wrapper for the GIN latent variable causal discovery algorithm in causal-learn
+"""
+
+from pandas import DataFrame
+
+from causallearn.search.HiddenCausal.GIN.GIN import GIN as GIN_
+from pywhy_graphs import CPDAG
+
+
+class GIN:
+    """Wrapper for GIN in the causal-learn package.
+
+    Parameters
+    ----------
+    indep_test_method : str
+        The method to use for testing independence, by default "kci"
+    alpha : float
+        The significance level for independence tests, by default 0.05
+
+    Attributes
+    ----------
+    graph_ : CPDAG
+        The estimated causal graph.
+    causal_learn_graph : CausalGraph
+        The causal graph object from causal-learn.
+    causal_ordering : list of str
+        The causal ordering of the variables.
```

Comment:
Is this the topological order?

We should document and explain this better for someone who doesn't know these terms.

Reply:
I'm not even sure what this ordering would be used for or if we need it.  I'm going to remove.

Reply:
[9e822bf](https://github.com/py-why/dodiscover/pull/72/commits/9e822bfd455d4582f983c2656ee60b1d5fb86b90)

Reply:
Yeah skimming the paper it seems like it's just topological order which we can extract from the graph